runtime/web_worker.rs

```diff
@@ -373,6 +376,10 @@ impl WebWorker {
         Ok(())
       })
       .build();
+    let create_cache = options.cache_storage_dir.map(|storage_dir| {
+      let create_cache_fn = move || SqliteBackedCache::new(storage_dir.clone());
+      CreateCache(Arc::new(Box::new(create_cache_fn)))
```

Comment:
This seems off, if you use `Arc` you shouldn't need to box it first

Reply:
Updated - thanks!